gen/sympa

```diff
@@ -25,29 +25,23 @@ our $A_USER_STATUS;                      *A_USER_STATUS =                     \'
 our $A_RESOURCE_MAILING_LIST_NAME;       *A_RESOURCE_MAILING_LIST_NAME =      \'urn:perun:resource:attribute-def:def:mailingListName';
 
 
-my $mailinglistStruc = {};  # $mailinglistStruc->{"mailing list name"}->{"user's e-mail"}->{A_USER_*} 
-my $mailinglistAdminMailStruc = {};  # $mailinglistAdminMailStruc->{"mailing list name"}->{"admin's e-mail"} 
-
+my $mailinglistStruc = {};  # $mailinglistStruc->{"mailing list name"}->{"user's e-mail"}->{"user's name"}
 
 my $mailinglistsDirectory = "$DIRECTORY/mailinglists";
 mkdir $mailinglistsDirectory or die "Can't mkdir $mailinglistsDirectory: $!";
 
 
-my @membersAttributes;
-my @resourcesData = $data->getChildElements;
-foreach my $rData (@resourcesData) {
-    my %resourceAttributes  = attributesToHash $rData->getAttributes;
-    local *FILE;
-
-    my @membersData = $rData->getChildElements;
+foreach my $resourceId ($data->getResourceIds()) {
+	my $listName = $data->getResourceAttributeValue(resource => $resourceId, attrName => $A_RESOURCE_MAILING_LIST_NAME);
 
-    my $listName = "$resourceAttributes{$A_RESOURCE_MAILING_LIST_NAME}";
-
-    foreach my $memberAttributes (dataToAttributesHashes @membersData) {
+	for my $memberId ($data->getMemberIdsForResource(resource => $resourceId)) {
         #list only VALID members except if allowNonvalidUsers is true
```

Comment:
I'ts not your change, but this comment doesn't make sense and its probably copy-paste from original mailman script. We do not check any resource attribute to allow non-valid users, script forces usage of only valid users by the next command. Please remove the comment. Same goes for `sympa_cesnet` version of the service.

Reply:
Removed.